tests/test_sandbox/test_ecs_fargate.py

```
@@ -1396,3 +1379,259 @@ def test_split_keeps_unrelated_env_in_stable(self):
+        with patch.object(ImageBuilder, "_mirror_and_push"):
+            ImageBuilder.ensure_image_mirrored(cfg=cfg, source_image=self._SRC, digest=self._DIGEST)
+
+        assert ecs_fargate.ImageBuilder._build_semaphore_size == 3
+
+    def test_image_mirror_mode_is_reachable_from_yaml(self):
+        """The YAML schema forbids extra keys — the field must exist there too."""
+        from nemo_evaluator.config.sandboxes import EcsFargateSandbox
+
+        assert EcsFargateSandbox(type="ecs_fargate").image_mirror_mode == "off"
+        assert EcsFargateSandbox(type="ecs_fargate", image_mirror_mode="codebuild").image_mirror_mode == "codebuild"
+        with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)